fabric/bin/lib/common_ledger.sh

```diff
@@ -112,15 +112,32 @@ ledger_init() {
     try rm -rf ${FABRIC_STATE_DIR}/*
 
     # 2. start orderer
+    # try to create orderer-system-channel genesisblock
+    para
+    echo "[+] Creating a genesisblock for orderer-system-channel......."
+    para
+    try ${CONFIGTXGEN_CMD} -outputBlock genesisblock -profile SampleDevModeSolo -channelID orderer-system-channel
+    sleep 1
+
     ORDERER_GENERAL_GENESISPROFILE=SampleDevModeSolo ${ORDERER_CMD} 1>${ORDERER_LOG_OUT} 2>${ORDERER_LOG_ERR} &
```

Comment:
here we also would have to change solo to raft (see above comment)

Reply:
As discussed on 2020-03-26, conversion to raft is moved out to separate PR. I created a corresponding issue #317